app/models/concerns/taxonomix.rb

```diff
@@ -3,12 +3,14 @@ module Taxonomix
   include DirtyAssociations
 
   included do
-    taxonomy_join_table = "taxable_taxonomies"
-    has_many taxonomy_join_table, :dependent => :destroy, :as => :taxable
-    has_many :locations,     :through => taxonomy_join_table, :source => :taxonomy,
-             :conditions => "taxonomies.type='Location'", :validate => false
-    has_many :organizations, :through => taxonomy_join_table, :source => :taxonomy,
-             :conditions => "taxonomies.type='Organization'", :validate => false
+    taxonomy_join_table = :taxable_taxonomies
+    has_many taxonomy_join_table.to_sym, :dependent => :destroy, :as => :taxable
+    has_many :locations, -> {where("taxonomies.type='Location'")},
```

Comment:
Can the where() clause use a hash `"taxonomies.type" => "Location"` rather than a string condition?  ditto below.